src/components/Search/Search.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SearchButton from './SearchButton.js';
+import TablTitle from './SearchHeading.js';
```

Comment:
This is a bit confusing to me - if you're importing a component, I would expect the variable to be named the same as the component name

Reply:
Yeah, you're right, I rename it